### PR TITLE
Add automated CI testing on Hera

### LINKED ISSUE #10: Add CI testing on Hera
Create a cronjob on Hera that will check for the label `hera-RT` and if it is on a pull request, check out, build, test said pull request and report back on success/fail.

### PR DESCRIPTION
**Pull request by CoryMartin-NOAA**
Closes #10 

This is building on top of PR #12  but adds Hera specific things.
Doing this now for testing since Orion is down today.

### CODE REVIEW

**Comment by guillaumevernieres** [comment]
> Automated Pull Request Testing Results: Machine: hera
> 
> ```
> Start: Wed Mar 23 16:09:40 UTC 2022 on hfe02
> ---------------------------------------------------
> Build:                                 *SUCCESS*
> Build: Completed at Wed Mar 23 16:53:34 UTC 2022
> ---------------------------------------------------
> Tests:                                 *SUCCESS*
> Tests: Completed at Wed Mar 23 16:53:37 UTC 2022
> Tests: 
> ```

I have to say, that is pretty darn cool! Thanks for doing the heavy lifting @CoryMartin-NOAA !

File: modulefiles/GDAS/hera.lua [hunk 1]
```diff
@@ -1,24 +1,65 @@
...
+load("atlas/ecmwf-0.24.1")
+load("nco/4.9.1")
+
+load("pybind11/2.7.0")
+load("json/3.9.1")
+load("json-schema-validator/2.1.0")
+
+load("hpc")
+load("miniconda3")
+load("gdasapp")
```
Comment (aerorahul) [inline modulefiles/GDAS/hera.lua]: versions.

File: modulefiles/GDAS/hera.lua [hunk 2]
```diff
@@ -1,24 +1,65 @@
--- NOAA RDHPCS Hera Modulefile for UFS-DA
 help([[
+Load environment for running JEDI applications with Intel compilers and MPI.
```
Comment (aerorahul) [inline modulefiles/GDAS/hera.lua]: GDASApp

**Review by aerorahul — DISMISSED** [review]
Looks good. 1 or 2 minor comments.

**Review by guillaumevernieres — DISMISSED** [review]
:+1:

**Review by guillaumevernieres — APPROVED** [review]

**Pull request merged**